scripts/misc/cpplint_ci.sh

```diff
@@ -35,6 +35,9 @@ else
   printf "Currently in %s branch. Running cpplint.\n" "$CURRENT_BRANCH"
 fi
 
+git branch --all
+git fetch origin master
+git branch --all
```

Comment:
I guess we don't need it

Reply:
38 and 40th lines are for more verbose logging. 39 line is mandatory.